verification/curator-service/api/test/sources.test.ts

```diff
@@ -3,6 +3,15 @@ import app from '../src/index';
 import mongoose from 'mongoose';
 import request from 'supertest';
 
+jest.mock('../src/clients/aws-events-client', () => {
+    const mockPutRule = jest
+        .fn()
+        .mockResolvedValue('arn:aws:events:fake:event:rule/name');
+    return jest.fn().mockImplementation(() => {
```

Comment:
you know me, always one-lineifying... does this work, or do the braces mess it up?

return jest.fn().mockImplementation(() => { putRule: mockPutRule});

Reply:
Unfortunately we can't. This needs to be a HOF, if you believe [Jest docs](https://jestjs.io/docs/en/es6-class-mocks#calling-jestmockdocsenjest-objectjestmockmodulename-factory-options-with-the-module-factory-parameter).